R/arrange.R

```diff
@@ -59,26 +59,37 @@
 #'   help page under the `Default locale` section.
 #' @family single table verbs
 #' @examples
-#' arrange(mtcars, cyl, disp)
-#' arrange(mtcars, desc(disp))
+#' # Load the {palmerpenguins} package (R < 4.5.0)
+#' library(palmerpenguins)
 #'
-#' # grouped arrange ignores groups
-#' by_cyl <- mtcars %>% group_by(cyl)
-#' by_cyl %>% arrange(desc(wt))
+#' # Arrange by increasing order of bill depth
+#' arrange(penguins, bill_depth_mm)
+#'
+#' # Arrange by increasing order of bill depth and body mass
+#' arrange(penguins, bill_depth_mm, body_mass_g)
+#'
+#' # Arrange by increasing order of bill depth and decreasing order of body mass
+#' arrange(penguins, bill_depth_mm, desc(body_mass_g))
```

Comment:
```suggestion
#' penguins %>% arrange(bill_depth_mm)
#'
#' # Arrange by increasing order of bill depth and body mass
#' penguins %>% arrange(bill_depth_mm, body_mass_g)
#'
#' # Arrange by increasing order of bill depth and decreasing order of body mass
#' penguins %>% arrange(bill_depth_mm, desc(body_mass_g))
```

This is what Hadley means by

> I'd also suggest keeping the existing pipe based examples primary, since I don't think we want to get into that discussion now.